code/modules/client/preferences.dm

```diff
@@ -350,31 +382,37 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 				dat += "[TextPreview(flavor_text)]...<br>"
 			dat += "<br>"
 
-			if((species in list("Unathi", "Vulpkanin", "Tajaran"))) //Species that have head accessories.
+			if(species in list("Unathi", "Vulpkanin", "Tajaran")) //Species that have head accessories.
 				var/headaccessoryname = "Head Accessory"
 				if(species == "Unathi")
 					headaccessoryname = "Horns"
 				dat += "<br><b>[headaccessoryname]</b><br>"
 				dat += "<a href='?_src_=prefs;preference=headaccessory;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_headacc, 2)][num2hex(g_headacc, 2)][num2hex(b_headacc, 2)]'><table style='display:inline;' bgcolor='#[num2hex(r_headacc, 2)][num2hex(g_headacc, 2)][num2hex(b_headacc)]'><tr><td>__</td></tr></table></font> "
 				dat += "Style: <a href='?_src_=prefs;preference=ha_style;task=input'>[ha_style]</a><br>"
-
-				dat += "<br><b>Body Markings</b><br>"
+			if(species in list("Unathi", "Vulpkanin", "Tajaran", "Machine"))
+				var/markings = "Body Markings"
+				if(species == "Machine")

```

Comment:
did you seriously add snowflake checks for IPC's everywhere in here for "correctness"